### PR TITLE
docs(website): add meta for charset and responsive web

### PR DESCRIPTION
**Pull request by fzn0x**
## 📝 Description

those lines of code are quite important in the context of HTML and web development for ensuring proper behavior and appearance of web pages across different devices and browsers.

`<meta charset="utf-8" />`: This line specifies the character encoding for the HTML document. UTF-8 is a universal character set that includes virtually all characters and symbols from all writing systems around the world, plus many technical symbols. Using UTF-8 prevents issues with non-ASCII characters (like those in languages other than English or special characters) appearing incorrectly on the web page.

`<meta name="viewport" content="width=device-width, initial-scale=1" />`: This line is crucial for making web pages mobile-friendly. It tells the browser to match the width of the page to the screen-width of the device (which varies significantly between devices like phones, tablets, and desktops) and to start with a zoom level of 1 (no zoom). This is essential for responsive web design, ensuring that your site looks good on all devices by allowing pages to render properly according to the device's screen size.

In summary, including these meta tags in the <head> section of your HTML document is considered best practice for modern web development. They play a key role in ensuring that your web pages are accessible, readable, and visually consistent across different browsers and devices.

## ⛳️ Current behavior (updates)

Those lines is not included in the website example.

```html
          <meta charSet="utf-8" />
          <meta name="viewport" content="width=device-width, initial-scale=1" />
```

## 🚀 New behavior

Include new lines for meta tags in remix-guide example.

### CODE REVIEW

**Comment by segunadebayo** [comment]
This had been fixed in the meta settings of the remix guide
